archivy/models.py

```diff
@@ -19,11 +20,27 @@ class DataObj:
     def process_bookmark_url(self):
         try:
             url_request = requests.get(self.url).text
+        except Exception:
+            sys.stderr.write("Could not retrieve {url}".format(url=self.url))
```

Comment:
Could you use f-strings instead of `.format`? It's a minor change but allows an easier way to embed the variables.

It would be like this: 
```python
sys.stderr.write(f"Could not retrieve {self.url}")
```

If there's a specific reason .format is better here, please tell me!

Otherwise please implement this for the other error.

Reply:
Done, here as well as in the elastic_error_handling PR.